src/Frame.h

```diff
@@ -183,10 +187,27 @@ namespace openshot
 		/// Display the wave form
 		void DisplayWaveform();
 
-		/// Get magnitude of range of samples (if channel is -1, return average of all channels for that sample)
-		float GetAudioSample(int channel, int sample, int magnitude_range);
+		/// Get highest value within a range of samples
+		//
+		/// @since 0.2.8
```

Comment:
This PR also marks our first 'since' annotation in the docs. 'since' is a handy tool for API authors to let consumers know when something was introduced to the library, so that they don't have to _guess_ whether they're using code that's not compatible with older versions.

I browbeat the CMake developers into adding since annotations to the CMake docs (and submitted PRs to add plenty of them myself), and it's made a world of difference in the usability of both the docs, and CMake's config language itself.